content/site.json

```diff
@@ -17,7 +17,8 @@
       }
     ],
     "warning": "We are in beta. Please familiarize yourself with [the market](https://oceanprotocol.com/technology/marketplaces), [the risks](https://blog.oceanprotocol.com/on-staking-on-data-in-ocean-market-3d8e09eb0a13), and the [Terms of Use](/terms).",
-    "warningPolygon": "Polygon/Matic EVM support is in early stages. Use the Matic bridge to [get mOCEAN](https://docs.oceanprotocol.com/tutorials/polygon-bridge/).",
+    "warningPolygon": "Polygon/Matic EVM support is in early stages.",
```

Comment:
Think this is now a good time to clean this content structure up a bit so content keys match component names, and we do not continue to add more and longer keys. 

Maybe like so:

```json
warnings: {
  "main": "We are in beta..."
  "polygonPublish": "Only republish data sets..."
},
"announcements": {
  "main": "Ocean Market is available on Polygon..."
  "polygon": "Polygon/Matic EVM support..."
}
```

This way we keep the main warning & announcement generic and are not forced to change keys in the future when we change messages